internal/engine/container_build_and_deployer.go

```diff
@@ -21,6 +21,7 @@ type ContainerBuildAndDeployer struct {
 	// May also fall back to ibd for certain error cases.
 	ibd ImageBuildAndDeployer
 
+	// skipContainer if true, we don't do a container build, and instead do an image build.
```

Comment:
`// if true...`?

Reply:
yeah, `skipContainer` is a boolean. If it's set to true, we don't do a container build.

Reply:
ah sorry i mean idk why you have the name of the variable at the start of the comment!

Reply:
ha yes but also those are supposed to be complete sentences: `// skipContainer indicates whether to skip container build...` etc. nbd but, 🤷‍♀️